src/components/TodoListItem/TodoListItem.jsx

```diff
@@ -1,8 +1,10 @@
+import style from "./TodoListItem.module.css"; // Import the CSS module
+
 function TodoListItem({ id, title, onRemoveTodo }) {
   return (
-    <tr>
-      <td style={{ border: "1px solid black", padding: "8px" }}> {title} </td>
-      <td style={{ border: "1px solid black", padding: "8px" }}>
+    <tr className={style.listItem}>
+      <td className={style.td}>{title}</td>
+      <td className={style.td}>
         <center>
           <button type="button" onClick={() => onRemoveTodo(id)}>
```

Comment:
Is it the expected behavior that all buttons in the app will inherit this style? If yes, this is fine. If not. add a classname.

Reply:
Yes, I wanted to keep all button elements with the same style.